src/App.tsx

```diff
@@ -1,24 +1,217 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { UserWarning } from './UserWarning';
+import {
+  getTodos,
+  addTodo,
+  deleteTodo,
+  completedTodo,
+  updateTodoTitle,
+} from './api/todos';
+import { Todo } from './types/Todo';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+import { FilterBy } from './types/FilterBy';
+import { Notification } from './components/Notification';
+import { NewTodo } from './types/NewTodo';
+import { ErrorNotifications } from './types/ErrorNotifications';
 
-const USER_ID = 0;
+const USER_ID = 6259;
+let isToggled = false;
```

Comment:
what's the purpose of this variable? why did you create it outside of the component and change somewhere inside? you can faces with issues with such approach

Reply:
you didn't reply for the comment, check it one more time